### ISSUE TITLE
What can I do to prevent YOLO from saving my model in wandb?

### ISSUE THREAD

**Issue by Cestovatels** [ultralytics/ultralytics#20075]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I activated wandb in my YOLO code. I don't want it to send best.pt and last.pt models to wandby. I don't even want to send some things in the media folder. I couldn't figure out how to do this.
My code construction is as follows;

```py
import wandb
from ultralytics import YOLO, settings

settings.update({'wandb': True})
wandb.login(key="*****************...")

model = YOLO("Yolov8x.pt")
model.train(data="data.yaml", epochs=epochs, imgsz=imgsz, project="training_name", name="training_name", batch=batch, patience=patience, device=devices, overlap_mask=overlap_mask, fliplr=fliplr)
```
Thanks for answer.

I think I need to make a change in [wb.py](https://github.com/ultralytics/ultralytics/blob/477412cce9b7125a0e14b3ea84998c71217de0a0/ultralytics/utils/callbacks/wb.py#L137) but I don't understand what to change.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Cestovatels, thank you for reaching out and for your interest in Ultralytics 🚀! 

We recommend checking out the [Ultralytics Docs](https://docs.ultralytics.com/) for guidance on managing integrations like Weights & Biases (WandB). You can find useful [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) examples there, as well as answers to many common questions.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a ❓ Question, it’s helpful to provide as much detail as possible, including your environment setup, training logs, and any modifications you’ve made. For your specific query, you might want to explore customization options in the `wb.py` file or configuration settings to adjust WandB behavior. 

Join the Ultralytics community for further support or discussions:
- [Discord](https://discord.com/invite/ultralytics) 🎧 for real-time help
- [Discourse](https://community.ultralytics.com/) for detailed discussions
- [Subreddit](https://reddit.com/r/Ultralytics) to share knowledge with others

## Upgrade

Ensure that you're using the latest version of the `ultralytics` package along with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). Run the following command to upgrade:

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, it means all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify the correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response to assist you quickly. An Ultralytics engineer will review your question and provide further insights soon. Thank you for your patience! 😊

**Comment by Y-T-G** [comment]
You will have to edit the callback code and remove the lines:

https://github.com/ultralytics/ultralytics/blob/dbe2b5d4bb27803e150a7e5fea4b5d4449032690/ultralytics/utils/callbacks/wb.py#L156-L159

**Comment by Y-T-G** [comment]
You can uses `plots=False` in `model.train()`

**Comment by Y-T-G** [comment]
You can remove this

https://github.com/ultralytics/ultralytics/blob/dbe2b5d4bb27803e150a7e5fea4b5d4449032690/ultralytics%2Futils%2Fcallbacks%2Fwb.py#L149
